src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -50,6 +50,20 @@ class ConfigurationHelper implements Serializable {
         return mixin(stepConfiguration, filter, compatibleParameters)
     }
 
+    ConfigurationHelper convertToAbsolutePaths(Set pathConfigurationKeys){
+        if(pathConfigurationKeys){
+            //Copy configuration to not persist any changes done below
+            this.config = MapUtils.deepCopy(this.config)
```

Comment:
dumb question: does this introduce new cps issues? cf https://github.com/SAP/jenkins-library/issues/786